client/my-sites/themes/theme-download-card/index.jsx

```diff
@@ -0,0 +1,40 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import Gridicon from 'components/gridicon';
+import i18n from 'lib/mixins/i18n';
+
+var ThemeDownloadCard = React.createClass( {
+
+	propTypes: {
+		theme: React.PropTypes.string.isRequired,
+		href: React.PropTypes.string
+	},
+
+	render() {
+		var downloadURI = this.props.href || ( 'https://downloads.wordpress.org/theme/' + this.props.theme + '.zip' );
+		var downloadText = i18n.translate( 'This theme is available for download to be used on your {{a}}WordPress self-hosted{{/a}} installation.', {
```

Comment:
The [preferred style](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#let-and-const) is to use `const` (or `let` when necessary) over `var`. For example: https://github.com/Automattic/wp-calypso/blob/d5360c590fa32f32bc226761d043c64dd32fc13d/client/my-sites/themes/sheet.jsx#L73